aaelf64-morello/aaelf64-morello.rst

```diff
@@ -476,6 +476,11 @@ The following nomenclature is used in the descriptions of relocation operations:
   the second entry holds a platform-specific offset or pointer. The pair of
   pointer-sized entries will be relocated with ``R_MORELLO_TLSDESC(S+A)``.
 
+- ``Delta(S)`` if ``S`` is a normal symbol, resolves to the difference between the
```

Comment:
This is just a copy of what's in aaelf64.rst, right?

Reply:
Yes.